my_function error: ValueError. Inputs: (1, 'a'), {}
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
my_function ok
